prism-languages.js

```diff
@@ -111,6 +111,22 @@ const installPrismLanguages = (Prism) => {
       /\b0x[\da-f]{6}[\da-f]*\b/i,
     ]
   });
+
+  // Basic highlighter for ES spec and Torque grammar excerpts.
+  Prism.languages.grammar = {
+    // matches the `[...]` part in `ProductionName[...]`
+    'production-params': /(?<=[a-z])\[.*?\]/,
+    // matches `ProductionName`
+    'production-name': /\b[A-Z][A-Za-z_]*\b/,
+    // "but not" and "one of" are special human-readable words in
+    // the ES spec that shouldn't be formatted as literals
+    'skip': /but not|one of/,
+    // `opt` is an ES grammar keyword to be formatted as a subscript
+    // `list+` and `list*` are Torque grammar keywords
+    'keyword': /\bopt\b|\blist[+*]/,
+    // anything else - numbers, words, punctuation - is a literal
+    'literal': /\S+/,
+  };
 };
 
 module.exports = installPrismLanguages;
```

Comment:
Echo'ing my earlier comment: since it's now no longer just about languages, `installPrismLanguages` -> `installPrismExtras`

Reply:
and rename the file accordingly

Reply:
Oh, I guess this is a `language` technically from Prism's point of view. Never mind then :)

Reply:
> since it's now no longer just about languages

I'm not sure what you mean. Are you saying that grammar syntax is not a language, so we should rename the function?

If so, then in Prism terms, it's still a "language" as we still add it to `Prism.languages.*`, it still generates a class `language-grammar` and so on, so I'm a bit worried that the naming mismatch will be more confusing not less, as we're not adding any other extras aside from extending `languages`...